src/main/java/com/rebook/review/controller/ReviewController.java

```diff
@@ -43,23 +43,27 @@ public ResponseEntity<List<ReviewResponse>> getReviews(@PathVariable("bookId") L
         return ResponseEntity.ok(responses);
     }
 
-    /**
-     @PostMapping("/reviews")
-     public ResponseEntity<ReviewResponse> saveReview(@Valid @RequestBody final ReviewRequest reviewRequest){
-     Review savedReview = reviewService.save(reviewRequest);
-     ReviewResponse reviewResponse = ReviewResponse.of(savedReview);
-     return ResponseEntity.created(URI.create("/api/v1/reviews")).body(reviewResponse);
-     }
+    @PutMapping("/{reviewId}")
+    @Operation(summary = "Update a Review for a Book", description = "해당 책의 특정 리뷰를 수정한다.")
+    public ResponseEntity<ReviewResponse> updateReview(
+            @PathVariable("bookId") Long bookId,
+            @PathVariable("reviewId") Long reviewId,
+            @Valid @RequestBody final ReviewRequest reviewRequest) {
+        Review updatedReview = reviewService.update(bookId, reviewId, reviewRequest);
+        ReviewResponse reviewResponse = ReviewResponse.from(updatedReview);
+        return ResponseEntity.ok(reviewResponse);
+    }
+
+    @DeleteMapping("/{reviewId}")
+    @Operation(summary = "Delete a Review for a Book", description = "해당 책의 특정 리뷰를 삭제한다.")
+    public ResponseEntity<Void> deleteReview(
```

Comment:
```suggestion
    public ResponseEntity<Void> softDeleteReview(
```

논리적 삭제를 명시적으로 구분해주면 좋을 것 같습니다.